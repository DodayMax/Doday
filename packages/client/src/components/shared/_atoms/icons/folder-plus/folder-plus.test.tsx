import * as React from 'react';
import { shallow } from 'enzyme';
import { Icons } from '@shared';

describe('FolderPlus', () => {
  it('renders correctly without props', () => {
    const folderPlus = shallow(<Icons.FolderPlus />);

    expect(folderPlus).toMatchSnapshot();
  });
});
