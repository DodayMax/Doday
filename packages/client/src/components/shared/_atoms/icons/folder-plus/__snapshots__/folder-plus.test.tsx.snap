// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FolderPlus renders correctly without props 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <defs>
          <path
            d="M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z"
            id="folder-1"
          />
        </defs>,
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <mask
            fill="#fff"
            id="folder-2"
          >
            <use
              xlinkHref="#folder-1"
            />
          </mask>
          <g
            fill="#000"
            mask="url(#folder-2)"
          >
            <path
              d="M0 0h24v24H0z"
            />
          </g>
        </g>,
      ],
      "height": "20",
      "viewBox": "0 0 24 24",
      "width": "20",
      "xmlns": "http://www.w3.org/2000/svg",
      "xmlnsXlink": "http://www.w3.org/1999/xlink",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <path
            d="M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z"
            id="folder-1"
          />,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z",
            "id": "folder-1",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        "type": "defs",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />,
            <mask
              fill="#fff"
              id="folder-2"
            >
              <use
                xlinkHref="#folder-1"
              />
            </mask>,
            <g
              fill="#000"
              mask="url(#folder-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>,
          ],
          "fill": "none",
          "fillRule": "evenodd",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M0 0h24v24H0z",
              "fill": "none",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <use
                xlinkHref="#folder-1"
              />,
              "fill": "#fff",
              "id": "folder-2",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "xlinkHref": "#folder-1",
              },
              "ref": null,
              "rendered": null,
              "type": "use",
            },
            "type": "mask",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <path
                d="M0 0h24v24H0z"
              />,
              "fill": "#000",
              "mask": "url(#folder-2)",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            "type": "g",
          },
        ],
        "type": "g",
      },
    ],
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <defs>
            <path
              d="M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z"
              id="folder-1"
            />
          </defs>,
          <g
            fill="none"
            fillRule="evenodd"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <mask
              fill="#fff"
              id="folder-2"
            >
              <use
                xlinkHref="#folder-1"
              />
            </mask>
            <g
              fill="#000"
              mask="url(#folder-2)"
            >
              <path
                d="M0 0h24v24H0z"
              />
            </g>
          </g>,
        ],
        "height": "20",
        "viewBox": "0 0 24 24",
        "width": "20",
        "xmlns": "http://www.w3.org/2000/svg",
        "xmlnsXlink": "http://www.w3.org/1999/xlink",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <path
              d="M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z"
              id="folder-1"
            />,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M11 12V9h2v3h3v2h-3v3h-2v-3H8v-2h3zm10-7a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h6c1.12 0 1.833.475 2.549 1.379.048.06.261.337.313.402.158.195.19.219.14.219H21zm0 14V7h-9.005c-.719-.004-1.186-.34-1.69-.963-.069-.086-.29-.373-.323-.416C9.607 5.15 9.384 5 9 5H3v14h18z",
              "id": "folder-1",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          "type": "defs",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />,
              <mask
                fill="#fff"
                id="folder-2"
              >
                <use
                  xlinkHref="#folder-1"
                />
              </mask>,
              <g
                fill="#000"
                mask="url(#folder-2)"
              >
                <path
                  d="M0 0h24v24H0z"
                />
              </g>,
            ],
            "fill": "none",
            "fillRule": "evenodd",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "d": "M0 0h24v24H0z",
                "fill": "none",
              },
              "ref": null,
              "rendered": null,
              "type": "path",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <use
                  xlinkHref="#folder-1"
                />,
                "fill": "#fff",
                "id": "folder-2",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "xlinkHref": "#folder-1",
                },
                "ref": null,
                "rendered": null,
                "type": "use",
              },
              "type": "mask",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <path
                  d="M0 0h24v24H0z"
                />,
                "fill": "#000",
                "mask": "url(#folder-2)",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "d": "M0 0h24v24H0z",
                },
                "ref": null,
                "rendered": null,
                "type": "path",
              },
              "type": "g",
            },
          ],
          "type": "g",
        },
      ],
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
